amplipi/app.py

```diff
@@ -81,7 +81,17 @@
 
 app.add_exception_handler(NotAuthenticatedException, not_authenticated_exception_handler)
 
-logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+loghandler = logging.StreamHandler()
+loghandler.setLevel(logging.INFO)
```

Comment:
where do you use this?

Reply:
https://github.com/micro-nova/AmpliPi/pull/661#discussion_r1521765478
Nowhere
I admittedly read as far as "I was imagining something more along the lines of (link)" and just followed the link as a tutorial, forgot to come back to see you saying this was unnecessary for our needs